package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Node-RED application to use bert-tokenizer and bert-sentiment",
   "main": "",
   "scripts": {
-    "start": "concurrently \"cd bert-sentiment-tfjs && npm run start\" \"node node_modules/.bin/node-red -s settings.js flow.json\"",
+    "start": "concurrently \"node node_modules/bert-sentiment-tfjs/dist/server/index.js\" \"node node_modules/.bin/node-red -s settings.js flow.json\"",
```

Comment:
can the `node node_modules/.bin/node-red -s settings.js flow.json\"` be just `node-red -s settings.js flow.json\"`

Reply:
That will call the global node-red.

Reply:
supposedly, the `./node_modules/.bin` will be pre-append to your path